go-selinux/selinux.go

```diff
@@ -106,7 +106,7 @@ func verifySELinuxfsMount(mnt string) bool {
 		}
 		return false
 	}
-	if buf.Type != selinuxfsMagic {
+	if uint32(buf.Type) != uint32(selinuxfsMagic) {
```

Comment:
We could change constant definition for selinuxfsMagic to be uint32 instead of casting here.

Reply:
That was the way the original patch was, but we sill need to cast the buf.Type,  I moved it here so it matched the C code.